Essentials/Commands/GridModule.cs

```diff
@@ -97,7 +98,11 @@ public void List()
 
                 if (grid.BigOwners.Contains(id))
                 {
+                    var gridGPS = MyAPIGateway.Session?.GPS.Create(grid.DisplayName, ($"{grid.DisplayName} - {grid.GridSizeEnum} - {grid.BlocksCount} blocks"), grid.PositionComp.GetPosition(), true);
```

Comment:
This is fine, but a tip for the future: Since the GPS object you're creating here will only be used if the ShowPosition option is on, you should move the allocation into the conditional at L104. That way you save resources by not creating an object you don't need.